addons/web/static/src/js/views/kanban/kanban_column_progressbar.js

```diff
@@ -269,7 +268,7 @@ var KanbanColumnProgressBar = Widget.extend({
         var filter = this.$clickedBar.data('filter');
         this.activeFilter = (this.activeFilter === filter ? false : filter);
         this._notifyState();
-        this._render();
+        this.trigger_up('load_column_active_filter');
```

Comment:
is the below handler dead (_onProgressBarParentClick) ?

Reply:
Yes it is